packages/ui-components/webpack.config.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 /* globals module, __dirname */
 const path = require("path");
-const WebpackBar = require("webpackbar");
```

Comment:
It got in the way of watch task reporting and spewed way too many lines. I can put it back if we want, but 🤷